extensions/nitrite/runtime/src/main/resources/META-INF/native-image/serialization-config.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Maybe, we can have a follow up issue to move the config to the deployment processor when/if https://github.com/quarkusio/quarkus/issues/14530 is implemented ?

Reply:
Yes, I didn't mention it, but the issue already exists: https://github.com/apache/camel-quarkus/issues/2255

Reply:
Great :+1: